website/blog/2024-02-28-maci-v1.2.0.md

```diff
@@ -0,0 +1,121 @@
+3. Fixing bugs and community-reported issues.
+
+### Library Updates
+
+MACI has relied on custom code and dependencies since its initial implementation, notably from repositories by one of the original MACI developers, [Koh Wei Jie](https://github.com/weijiekoh). We felt that MACI could benefit from a dependency refresh, so we've shifted towards using actively maintained open-source libraries, such as [circomkit](https://github.com/erhant/circomkit) and [zk-kit](https://github.com/privacy-scaling-explorations/zk-kit).
+
+Circomkit has become our go-to for circuit-related tasks, such as compiling circuits, generating test zkeys, and unit tests.
+
+We've moved reusable circuit logic, like our [Poseidon permutation](https://github.com/privacy-scaling-explorations/zk-kit/tree/main/packages/poseidon-cipher) encryption and decryption code, into zk-kit. This not only benefits MACI through more circuit usage and testing but also supports wider community adoption.
+
+These efforts are part of a broader initiative at [Privacy and Scaling Explorations (PSE)](https://pse.dev/) to foster open-source development and contribute to public goods. By aligning MACI with these values, we aim to enhance its utility and encourage collaborative growth. In the coming months, we're committed to extracting more of our circuit logic for broader use and contributing to projects that align with our mission. These efforts will continue to be tracked [here](https://github.com/privacy-scaling-explorations/zk-kit/issues/131).
+
+### Code Refactor
+
+We've undergone extensive efforts to clean up MACI's code:
+
+- Removed dead and redundant code.
+- Split monolithic files into smaller, logically structured ones.
+- Improved documentation with detailed code comments and tools like ([TypeDoc](https://typedoc.org/) and [solidity-docgen](https://github.com/OpenZeppelin/solidity-docgen)) for automatic doc generation.
+- Enforced strong type safety on the TypeScript components.
+- Optimized and modularized the smart contract code.
+- Extended the test suites.
+
+These improvements aim to simplify the onboarding process for new developers, ensuring they can easily navigate and effectively utilize MACI.
+
+## New Features
+
+### Flexible Voting Strategies
+
+After years of built-in quadratic voting (QV) in MACI, we now support non-quadratic voting polls. This opens up the door to potential integrations with a wider variety of projects, such as DAO governance applications.
+
+The new version of the Tally circuit (specific for non-QV), has reduced constraints, enabling a quicker proof generation process for vote tallying.
+
+We invite projects interested in leveraging this secure, on-chain voting mechanism to reach out for potential integrations. We hope to continue to expand support for additional voting methodologies, so please let us know your project's needs!
+
+### New Gatekeepers (EAS and Hats Protocol)
```

Comment:
I think the **Hats Protocol** is not yet merged?

Reply:
Good point! we could say it's in progress for now and include in 1.2.1?